create_jenkins_job.py

```diff
@@ -46,6 +46,14 @@
     [os.path.join(os.path.abspath(os.path.dirname(__file__)), 'job_templates')]
 
 
+def nonnegative_int(inval):
+    ret = int(inval)
```

Comment:
You probably want to catch the possible exception here too.

Reply:
Fixed in f13827d